README.md

```diff
@@ -170,6 +170,7 @@ The following extensions can be added into MakeCode by copying the GitHub URL an
 - [ESP-01](https://github.com/51bit/esp01) - Control an ESP8266 module via serial AT commands.
 - [TCS3200](https://github.com/DoraLC/pxt-tcs3200-color-sensor) - Control a TCS3200 color sensor.
 - [IR](https://github.com/lioujj/pxt-IR) - Control IR (infrared) transmitter/receiver modules.
+- [DSTemp Alpha](https://github.com/bsiever/microbit-dstemp-alpha) - Read temperature from DS18B20 sensor. Supports multiple sensors. - Alpha
```

Comment:
```suggestion
- [DSTemp Alpha](https://github.com/bsiever/microbit-dstemp-alpha) - MakeCode extension to read temperature from DS18B20 sensor, supporting multiple sensors (alpha release).
```

Reply:
This change is no longer necessary as for the tweet the section will me automatically included.